database.sql

```diff
@@ -525,24 +534,17 @@ CREATE TABLE IF NOT EXISTS `item` (
 	 INDEX `uid_network_received` (`uid`,`network`,`received`),
 	 INDEX `uid_received` (`uid`,`received`),
 	 INDEX `uid_network_commented` (`uid`,`network`,`commented`),
-	 INDEX `uid_commented` (`uid`,`commented`),
 	 INDEX `uid_title` (`uid`,`title`),
 	 INDEX `uid_thrparent` (`uid`,`thr-parent`),
 	 INDEX `uid_parenturi` (`uid`,`parent-uri`),
 	 INDEX `uid_contactid_id` (`uid`,`contact-id`,`id`),
 	 INDEX `uid_contactid_created` (`uid`,`contact-id`,`created`),
-	 INDEX `gcontactid_uid_created` (`gcontact-id`,`uid`,`created`),
```

Comment:
Thank you for cleaning this up. Extra indexes causes more index data being loaded. This change will make the database footprint slimmer.